pkg/sql/catalog/descpb/privilege.go

```diff
@@ -67,18 +63,18 @@ func (p PrivilegeDescriptor) findUserIndex(user security.SQLUsername) int {
 	return -1
 }
 
-// findUser looks for a specific user in the list.
+// FindUser looks for a specific user in the list.
 // Returns (nil, false) if not found, or (obj, true) if found.
-func (p PrivilegeDescriptor) findUser(user security.SQLUsername) (*UserPrivileges, bool) {
+func (p PrivilegeDescriptor) FindUser(user security.SQLUsername) (*UserPrivileges, bool) {
```

Comment:
This and the methods below were made public for access from `catprivileges` in particular.